integration-libs/digital-payments/schematics/add-digital-payments/index.ts

```diff
@@ -0,0 +1,59 @@
+import {
+  chain,
+  noop,
+  Rule,
+  SchematicContext,
+  Tree,
+} from '@angular-devkit/schematics';
+import {
+  addLibraryFeature,
+  addPackageJsonDependenciesForLibrary,
+  CLI_DIGITAL_PAYMENTS_FEATURE,
+  LibraryOptions as SpartacusDigitalPaymentsOptions,
+  readPackageJson,
+  shouldAddFeature,
+  SPARTACUS_CHECKOUT,
+  CLI_CHECKOUT_FEATURE,
+  SPARTACUS_DIGITAL_PAYMENTS,
+  validateSpartacusInstallation,
+} from '@spartacus/schematics';
+import { peerDependencies } from '../../package.json';
+import { DIGITAL_PAYMENTS_FOLDER_NAME, DIGITAL_PAYMENTS_MODULE, DIGITAL_PAYMENTS_MODULE_NAME,
+} from '../constants';
+
+export function addDigitalPaymentsFeature(options: SpartacusDigitalPaymentsOptions): Rule {
+  return (tree: Tree, _context: SchematicContext) => {
+    const packageJson = readPackageJson(tree);
+    validateSpartacusInstallation(packageJson);
+
+    return chain([
+      addPackageJsonDependenciesForLibrary(peerDependencies, options),
+
+      shouldAddFeature(CLI_DIGITAL_PAYMENTS_FEATURE, options.features)
+        ? addDigitalPayments(options)
+        : noop(),
+    ]);
+  };
+}
+
+function addDigitalPayments(options: SpartacusDigitalPaymentsOptions): Rule {
+  return addLibraryFeature(options, {
+    folderName: DIGITAL_PAYMENTS_FOLDER_NAME,
+    moduleName: DIGITAL_PAYMENTS_MODULE_NAME,
+    rootModule: {
+      importPath: SPARTACUS_DIGITAL_PAYMENTS,
+      name: DIGITAL_PAYMENTS_MODULE,
+      content: `${DIGITAL_PAYMENTS_MODULE}.forRoot()`,
+    },
+    featureModule: {
+      importPath: SPARTACUS_DIGITAL_PAYMENTS,
+      name: DIGITAL_PAYMENTS_MODULE,
+    },
+    dependencyManagement: {
```

Comment:
Also section for i18n needs to be added (translations)

Reply:
done